### PR TITLE
finished creating problems for organisers, created the base for modifying problems

### PR DESCRIPTION
**Pull request by filipides1**

### CODE REVIEW

**Review by fsipic — APPROVED** [review]

File: src/services/problem.service.ts [hunk 1]
```diff
@@ -0,0 +1,88 @@
+import requests from "../requests"
+
+
+const createProblem = async (
```
Comment (TeoMatosevic) [inline src/services/problem.service.ts:4]: Create interfaces for input types (see how we did it)

File: src/services/problem.service.ts [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+	const response = await requests.post("/problems", formData, {
+		headers: {
+			ContentType: "multipart/form-data",
+		},
+	})
+	console.log(response)
+	return response
+}
+
+const modifyProblem = async (
```
Comment (TeoMatosevic) [inline src/services/problem.service.ts:38]: Again, interface

File: src/App.tsx [hunk 3]
```diff
@@ -13,6 +13,7 @@ import { Role } from "./Models"
 import { Navbar } from "./components"
 import { AuthProvider } from "./context/AuthContext"
 
+
```
Comment (TeoMatosevic) [inline src/App.tsx:16]: Remove this line

File: src/pages/CreateProblemPage.tsx [hunk 4]
```diff
@@ -0,0 +1,355 @@
+import { Navbar } from "../components"
+import { useState, useRef, FormEvent, useEffect } from "react"
+import { Button } from "primereact/button"
+import { ProgressSpinner } from "primereact/progressspinner"
+import { InputText } from "primereact/inputtext"
+import { InputTextarea } from "primereact/inputtextarea"
+import { CreateProblem } from "../Models"
```
Comment (TeoMatosevic) [inline src/pages/CreateProblemPage.tsx:7]: Is this even used?

File: src/pages/EditProblemPage.tsx [hunk 5]
```diff
@@ -0,0 +1,393 @@
+import { Navbar } from "../components"
+import { useState, useRef, FormEvent, useEffect } from "react"
+import { Button } from "primereact/button"
+import { ProgressSpinner } from "primereact/progressspinner"
+import { InputText } from "primereact/inputtext"
+import { InputTextarea } from "primereact/inputtextarea"
+import { ModifyProblem } from "../Models"
```
Comment (TeoMatosevic) [inline src/pages/EditProblemPage.tsx:7]: Is this even used?

File: src/pages/Organiser.css [hunk 6]
```diff
@@ -0,0 +1,8 @@
+.scrollbar-hide::-webkit-scrollbar {
```
Comment (TeoMatosevic) [inline src/pages/Organiser.css:1]: Remove this and move it to global.css

File: src/pages/CreateProblemPage.tsx (hunk 4 above)
Comment (filipides1) [reply src/pages/CreateProblemPage.tsx:7]: yes in line 22

File: src/pages/EditProblemPage.tsx (hunk 5 above)
Comment (filipides1) [reply src/pages/EditProblemPage.tsx:7]: yes in line 26

**Pull request merged**